VEHICLE/SetRandomTrains.md

```diff
@@ -8,6 +8,36 @@ ns: VEHICLE
 void SET_RANDOM_TRAINS(BOOL toggle);
 ```
 
+**Information**
+Enables spawning random trains on the preset tracks. However to be able to use this call you would first need to call at least two other calls to make it function properly.
+
+You would need to tell the game which tracks you want to be enabled (or disabled), and you would need to set the (random) spawn frequency. If you don't call at least those to functions this function won't do anything at all!
+
+**Required other functions:**
```

Comment:
This would probably be better off being 
```
### Related and useful natives
[SWITCH_TRAIN_TRACK](#_0xFD813BB7DB977F20)
[SET_TRAIN_TRACK_SPAWN_FREQUENCY](#_0x21973BBF8D17EDFA)
```
as to not include redundant information.